packages/sourcecred/src/api/instance/instance.js

```diff
@@ -47,5 +47,7 @@ export interface Instance extends ReadOnlyInstance {
   writeLedger(ledger: Ledger): Promise<void>;
 
   /** writes grain integration output */
-  writeGrainIntegrationOutput(result: GrainIntegrationResult): Promise<void>;
+  writeGrainIntegrationOutput(
+    result: $Shape<GrainIntegrationMultiResult>
```

Comment:
why shape? seems unnecessary since the GrainIntegrationOutput attribute is optional already

Reply:
I wanted to be unopinionated enough to accept the `GrainIntegrationResult`type.